v2/azure-arm.yaml

```
@@ -3683,6 +3683,30 @@ objectModelConfiguration:
         $export: true
         $supportedFrom: v2.10.0
     2024-03-01:
+      AzureFirewall:
+        $export: true
```

Prefer avoiding `$export`, I want to remove it.

```suggestion
        $exportAs: AzureFirewall
```